app/models/plan.rb

```diff
@@ -573,6 +611,14 @@ def owner
   end
 
   ##
+  # the owner and co-owners of the project
+  #
+  # @return [Users]
+  def owner_and_coowners
+    User.joins(:roles).where("roles.plan_id = ? AND roles.access >= ?", self.id, 14)
```

Comment:
Could you explain to me why just 14? If the method is doing what I thinking... "give all the owners and co-owners of the current instance plan"... it should be:

- roles.plan_id = 'foo_id' && roles.access in (1,3,5,7,9,11,13,15) OR roles.access in (2,3,6,7,10,11,14,15)

Reply:
Well, we only have 5 available options actually stored in the DB (which is why I opted for 14 instead of all the possible values). We could probably just have used an Int enum for this column though. Since we only ever have 5 values in the DB:
```
Roles                                              Access           Nbr Recs
-----------------------------------------------------------------------------------
commenter                                          8                404
editor + commenter                                 12               1145
administrator + editor + commenter                 14               1532
creator + administrator + editor + commenter       15               18052
commenter + reviewer                               24               30
```

I found an even better solution though by digging through the gem's code (they're documentation isn't the best): `access_values_for(:flag)` It returns all of the values for the flag.
`Role.access_values_for(:reviewer, :commenter).min` returns 24
`Role.access_values_for(:creator).concat(Role.access_values_for(:administrator))` returns all of the values you referenced above